### PR TITLE
chore: visual refresh on tabs component

### PR DESCRIPTION
**Pull request by khamudom**
# Pull Request

## 📖 Description

different states on unselected tab (tab three)
before
image
after
image

different states on the selected tab (tab three)
before
image
after
image

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/src/tab/tab.styles.ts [hunk 1]
```diff
@@ -42,122 +49,119 @@ export const tabStyles: (
...
+        }
 
-    :host([disabled]) {
-        cursor: ${disabledCursor};
-        opacity: ${disabledOpacity};
-    }
+        :host([aria-selected="true"]),
+        :host(.vertical[aria-selected="true"]) {
+            z-index: 2;
+            background: ${accentFillRest};
```
Comment (bheston) [inline packages/web-components/fast-components/src/tab/tab.styles.ts:86]: Let's make this like listbox option or tree view item. Selected state should be the accent border with neutral fill.

File: packages/web-components/fast-components/src/tabs/index.ts [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+ * @public
+ */
+export class Tabs extends FoundationTabs {
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+
+        fillColor.setValueFor(this, neutralLayerFloating);
```
Comment (bheston) [inline packages/web-components/fast-components/src/tabs/index.ts:19]: This shouldn't be needed for tabs

**Review by bheston — CHANGES_REQUESTED** [review]

File: packages/web-components/fast-components/src/tab/tab.styles.ts (hunk 1 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/tab/tab.styles.ts]: updated

File: packages/web-components/fast-components/src/tabs/index.ts (hunk 2 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/tabs/index.ts]: file reverted

**Review by bheston — APPROVED** [review]

File: packages/web-components/fast-components/src/tab/tab.styles.ts [hunk 3]
```diff
@@ -42,122 +50,121 @@ export const tabStyles: (
...
-            :host([disabled]:hover) {
-                opacity: 1;
-                color: ${SystemColors.GrayText};
-                background: ${SystemColors.ButtonFace};
-            }
-        `
+                :host([disabled]),
+                :host([disabled]:hover) {
+                    opacity: 1;
+                    border-color: ${SystemColors.GrayText};
```
Comment (robarbms) [inline packages/web-components/fast-components/src/tab/tab.styles.ts:164]:
```suggestion
                    border-color: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tabs.styles.ts [hunk 4]
```diff
@@ -38,36 +38,26 @@ export const tabsStyles: (
 
         .tablist {
             display: grid;
-            grid-template-rows: auto auto;
+            grid-template-rows: calc(${heightNumber} * 1px); auto;
```
Comment (robarbms) [inline packages/web-components/fast-components/src/tabs/tabs.styles.ts:41]:
```suggestion
            grid-template-rows: calc(${heightNumber} * 1px) auto;
```

File: packages/web-components/fast-components/src/tabs/tabs.styles.ts [hunk 5]
```diff
@@ -38,36 +38,26 @@ export const tabsStyles: (
...
             grid-template-columns: auto;
+            grid-column-gap: 8px;
```
Comment (robarbms) [inline packages/web-components/fast-components/src/tabs/tabs.styles.ts:43]: Should this be something based off of density or designUnit?

File: packages/web-components/fast-components/src/tabs/tabs.styles.ts [hunk 6]
```diff
@@ -86,13 +76,12 @@ export const tabsStyles: (
             display: grid;
             grid-template-rows: auto;
             grid-template-columns: auto 1fr;
+            grid-row-gap: 8px;
```
Comment (robarbms) [inline packages/web-components/fast-components/src/tabs/tabs.styles.ts:79]: Same as above, should this be based off of density or designUnit?

File: packages/web-components/fast-components/src/tabs/tabs.styles.ts (hunk 5 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/tabs/tabs.styles.ts]: updated

File: packages/web-components/fast-components/src/tabs/tabs.styles.ts (hunk 6 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/tabs/tabs.styles.ts]: updated

File: packages/web-components/fast-components/src/tabs/tabs.styles.ts (hunk 4 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/tabs/tabs.styles.ts]: updated

File: packages/web-components/fast-components/src/tab/tab.styles.ts (hunk 3 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/tab/tab.styles.ts]: updated

**Review by robarbms — APPROVED** [review]

**Pull request merged**